README.md

```diff
@@ -2,16 +2,24 @@
 
 ## Example Command
 
+[![build](https://github.com/KevenDuverglas/keven-cellveyor/actions/workflows/build.yml/badge.svg)](https://https://github.com/KevenDuverglas/keven-cellveyor/actions/workflows/build.yml)
```

Comment:
Hi @KevenDuverglas, you need to make sure that the documentation does not link to a badge that is inside of a repository in your account.